src/sass/_componentSassImporter.scss

```diff
@@ -25,6 +25,7 @@ them here to have them available to the app
 @import 'orderCart';
 @import 'orderCartView';
 @import 'orderTnrisDataForm';
+@import '_surveyModal';
```

Comment:
i'm pretty certain you don't need the prefix underscore when importing your sass file. can we remove it for consistency?